### PR TITLE
Update README.md

### LINKED ISSUE #1703: README description for running local tests is out of date
### Bug Report
The description in the README for running tests locally is out of date and should be updated.

The description should be updated to:
```
xcodebuild build-for-testing -project SmartDeviceLink-iOS.xcodeproj -destination platform=iOS Simulator,name=iPhone 11,OS=13.5 -scheme SmartDeviceLink

set -o pipefail && xcodebuild test-without-building -project SmartDeviceLink-iOS.xcodeproj -destination platform=iOS Simulator,name=iPhone 11,OS=13.5 -scheme SmartDeviceLink -configuration Debug ONLY_ACTIVE_ARCH=NO RUN_CLANG_STATIC_ANALYZER=NO GCC_INSTRUMENT_PROGRAM_FLOW_ARCS=YES GCC_GENERATE_TEST_COVERAGE_FILES=YES ENABLE_TESTABILITY=YES
```

### LINKED ISSUE #1726: Update README
### Bug Report
Update the README with updated information about developing for the library and remove content that is duplicated in the guides.

### PR DESCRIPTION
**Pull request by joeljfischer**
Fixes #1703, #1726 

This PR is **not ready** for review.

### Risk
This PR makes **no** API changes.

### Testing Plan
- [ ] I have verified that I have not introduced new warnings in this PR (or explain why below)
- [ ] I have run the unit tests with this PR
- [ ] I have tested this PR against Core and verified behavior (if applicable, if not applicable, explain why below).

#### Unit Tests
n/a Only a README update

#### Core Tests
n/a Only a README update

Core version / branch / commit hash / module tested against: n/a
HMI name / version / branch / commit hash / module tested against: n/a

### Summary
Update the README with new and more relevant information.

### Tasks Remaining:
none

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -156,25 +80,23 @@ cd ../
...
-You can optionally pipe the result to [xcpretty](https://github.com/supermarin/xcpretty).
+set -o pipefail && xcodebuild test-without-building -project SmartDeviceLink-iOS.xcodeproj -destination platform=iOS Simulator,name=iPhone 11,OS=13.5 -scheme SmartDeviceLink -configuration Debug ONLY_ACTIVE_ARCH=NO RUN_CLANG_STATIC_ANALYZER=NO GCC_INSTRUMENT_PROGRAM_FLOW_ARCS=YES GCC_GENERATE_TEST_COVERAGE_FILES=YES ENABLE_TESTABILITY=YES
+```
 
-#### Lock Screen Screenshot Tests
-We run some additional tests using [FBSnapshotTestCase](https://github.com/uber/ios-snapshot-test-case). These tests generate the lock screen view controller and compare it to generated screenshots. By default, the generated screenshots use the iPhone 11 / iPhone XR simulator. If you run unit tests against those simulators, the tests should pass.
+##### Lock Screen Screenshot Tests
+We run some additional tests using [FBSnapshotTestCase](https://github.com/uber/ios-snapshot-test-case). These tests generate the lock screen view controller and compare it to generated screenshots. By default, the generated screenshots use the iPhone 11 simulator. If you run unit tests against those simulators, the tests should pass.
```
[the remark this answers is not in the document]
Comment (joeljfischer) [reply README.md:89]: Updated slightly modified

File: README.md [hunk 2]
```diff
@@ -7,144 +7,68 @@
...
+```bash
 lipo -info SmartDeviceLink.framework/SmartDeviceLink
 ```
 
-### Reference Documentation
+### Getting Help
+If you have questions, first view our guides on the [SmartDeviceLink website](https://smartdevicelink.com/en/guides/iOS/getting-started/installation/).
 
-You can find the latest reference documentation on [SmartDeviceLink.com Documentation](https://smartdevicelink.com/en/docs/iOS/master/), and view guides on [SmartDeviceLink.com Guides](https://smartdevicelink.com/en/guides/iOS/getting-started/installation/).
+If you see a bug, please post an issue on the appropriate repository. Please see the [contribution guidelines](https://github.com/smartdevicelink/sdl_ios/blob/master/CONTRIBUTING.md) before proceeding. If you need general assistance, or have other questions, you can [sign up](http://slack.smartdevicelink.org) for the [SDL Slack](https://smartdevicelink.slack.com) and chat with other developers and the maintainers of the project.
```
[the remark this answers is not in the document]
Comment (joeljfischer) [reply README.md:45]: Updated a modified version. The sign up link was incorrect and the change for the slack link was correct originally.

File: README.md [hunk 3]
```diff
@@ -7,144 +7,68 @@
...
-### Reference Documentation
+### Getting Help
+If you have questions, first view our guides on the [SmartDeviceLink website](https://smartdevicelink.com/en/guides/iOS/getting-started/installation/).
 
-You can find the latest reference documentation on [SmartDeviceLink.com Documentation](https://smartdevicelink.com/en/docs/iOS/master/), and view guides on [SmartDeviceLink.com Guides](https://smartdevicelink.com/en/guides/iOS/getting-started/installation/).
+If you see a bug, please post an issue on the appropriate repository. Please see the [contribution guidelines](https://github.com/smartdevicelink/sdl_ios/blob/master/.github/CONTRIBUTING.md) before proceeding. If you need general assistance, or have other questions, you can [sign up](http://slack.smartdevicelink.com) for the [SDL Slack](https://smartdevicelink.slack.com) and chat with other developers and the maintainers of the project.
 
-### Getting Help
+### Example Apps
+If you'd like to use the Hello SDL example apps and have Cocoapods installed, you can use `pod try SmartDeviceLink`, alternately, you can clone or download the project.
```
Comment (NicoleYarroch) [inline README.md:48]:
```suggestion
The library contains two example apps: one written in Objective-C and one in Swift. If you have CocoaPods installed, you can easily run one of the example apps by executing `pod try SmartDeviceLink` in your terminal. Alternatively, you can clone or download the project, but before attempting to build and run the example apps you must follow the [installation steps](#preparing-for-development).
```

File: README.md [hunk 4]
```diff
@@ -7,144 +7,68 @@
...
+3. We welcome PRs for bug fixes! If there's a confirmed issue for a bug, we would welcome a bug fix. The PR will require a review from a project maintainer before it can be merged.
+4. PRs for features require additional planning. Features must be approved by the SDLC using the [Evolution Process](https://github.com/smartdevicelink/sdl_evolution) before an issue can be opened. Furthermore, a feature must be approved to be developed for a release by the SDLC before a PR can be opened. Finally, a PR cannot be merged if the underlying feature requires or should have alignment across multiple projects until PRs for all those projects are ready for review.
+
+#### Preparing for Development
+To prepare the library for development, you will need to take a few steps:
+
+1. Clone this repository.
+1. Install submodules with `git submodule init` and `git submodule update`.
+1. Install [testing dependencies](#running-tests).
+1. Ensure that you can run the [Hello SDL test app](#example-apps).
```
Comment (NicoleYarroch) [inline README.md:66]:
```suggestion
1. Ensure that you can run the [Example test apps](#example-apps).
```

File: README.md [hunk 5]
```diff
@@ -7,144 +7,68 @@
...
-<string>com.smartdevicelink.prot4</string>
-<string>com.smartdevicelink.prot3</string>
-<string>com.smartdevicelink.prot2</string>
-<string>com.smartdevicelink.prot1</string>
-<string>com.smartdevicelink.prot0</string>
-<string>com.ford.sync.prot0</string>
-<string>com.smartdevicelink.multisession</string>
-</array>
-```
+There are additional example apps available [in the example organization](https://github.com/smartdevicelink-examples/), these require Cocoapods to install dependencies.
```
Comment (NicoleYarroch) [inline README.md:50]:
```suggestion
There are additional example apps available [in the example organization](https://github.com/smartdevicelink-examples/), these require CocoaPods to install dependencies.
```

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (joeljfischer) [reply README.md:48]: Yeah, I wasn't sure what the call them.

Comment (joeljfischer) [reply README.md:48]: Updated but modified slightly

**Pull request merged**